project-base/storefront/components/Blocks/ConfirmationPage/ConfirmationPageContent.tsx

```diff
@@ -0,0 +1,43 @@
+import useTranslation from 'next-translate/useTranslation';
+import { ReactElement } from 'react';
+
+type ConfirmationPageContentProps = {
+    type: 'order-created' | 'payment-successful' | 'payment-failed';
+    content?: string;
+    AdditionalContent?: ReactElement;
+};
+
+export const ConfirmationPageContent: FC<ConfirmationPageContentProps> = ({ type, content, AdditionalContent }) => {
+    const { t } = useTranslation();
+
+    const Image = (() => {
+        if (type === 'order-created') {
+            return <img alt={t('Your order was created')} src="/public/frontend/images/sent-cart.svg" />;
+        } else if (type === 'payment-failed') {
+            return <img alt={t('Your payment was not successful')} src="/public/frontend/images/sent-cart.svg" />;
+        }
+        return <img alt={t('Your payment was successful')} src="/public/frontend/images/sent-cart.svg" />;
+    })();
+
+    const HeadingText = (() => {
+        if (type === 'order-created') {
+            return t('Your order was created');
+        } else if (type === 'payment-failed') {
+            return t('Your payment was not successful');
+        }
+        return t('Your payment was successful');
+    })();
+
+    return (
+        <div className="mt-16 mb-10 flex flex-col items-center justify-center lg:mt-16 lg:mb-24 lg:flex-row">
+            <div className="mb-0 w-40 lg:mr-32">{Image}</div>
+            <div>
+                <div className="h1 mb-3">{HeadingText}</div>
+                {!!content && (
+                    <div className="text-center lg:text-left" dangerouslySetInnerHTML={{ __html: content }} />
+                )}
+                {!!content && !!AdditionalContent && AdditionalContent}
```

Comment:
Why do we display this component at all if we don't have `content`? Can we add condition in parent component?

Reply:
So the point is to show something even after 5 minutes. The user has 5 minutes to display the sensitive content (with order details etc) and then he can display a very generic message afterwards.